content/doc/book/blueocean/dashboard.adoc

```diff
@@ -19,105 +19,78 @@ endif::[]
 
-Clicking on an item in the Pipelines list will display that item's
-link:../activity[Activity View].
+Selecting an item in the Pipelines list will display that item's link:../activity[Activity View].
 
 
 == Favorites list
 
-The *Favorites* list appears above the Dashboard's default <<pipelines-list>>
-when at least one Pipeline/item is present in your user's Favorites list.
+The *Favorites* list appears above the Dashboard's default <<pipelines-list>> when at least one Pipeline/item is present in your Favorites list.
 
-This list provides key information and actions for a core subset of your user's
-accessible items in the <<pipelines-list>>. This key information includes the
-current link:#run-status[run status] for an item and its repository's branch, as
-well as other details about the item's run, including the name of the branch,
-the initial part of the commit hash, and the last run time. This list also 
-includes clickable icons to run or re-run the item on the repository
-branch indicated.
+This list provides key information and actions for a core subset of your accessible items in the <<pipelines-list>>.
+This key information includes the current link:#run-status[run status] for an item and its repository's branch, the name of the branch, the initial part of the commit hash, and the last run time. 
+This list also includes options to run or re-run the item on the indicated branch.
 
-You should only add an item (or one of the repository's specific branches) to
-your Favorites list if you need to regularly examine that item's branch. Adding
-an item's specific branch to your Favorites list can be done
-through the item's link:../activity[Activity View].
+You should only add an item or branch to your Favorites list if it needs regular monitoring.
+Adding an item's specific branch to your Favorites list can be done through the item's link:../activity[Activity View].
 
-Blue Ocean automatically adds branches or PRs to this list when they contain a
-run that has changes authored by the current user.
+Blue Ocean automatically adds branches and PRs to this list when they include a run that contains any modifications you have performed.
 
-You can also manually remove items from your Favorites list by clicking on the
-solid "&#9733;" in this list. When the last item is removed from this list, the
-list is removed from the interface.
-
-Clicking on an item in the Favorites list will open the
-link:../pipeline-run-details[Pipeline run details] for the latest run on the
-repository branch or PR indicated.
+You can also manually remove items from your Favorites list by deselecting the solid &#9733; in this list.
+When there are no longer items in this list, the list is removed from the Dashboard.
+Selecting the favorite &#9733; for any item will bring the list back to your dashboard.
 
+Selecting an item in the Favorites list will open the link:../pipeline-run-details[Pipeline run details] for the latest run on the branch or PR indicated.
```

Comment:
Our bee write standards say to avoid future tense.
```suggestion
Selecting an item in the *Favorites* list opens the link:../pipeline-run-details[Pipeline run details] for the latest run on the branch or PR indicated.
```

Reply:
that's right, thanks for noting this!